src/react-components/src/components/01-atoms/Text/Headings/RNSectionTitle.tsx

```diff
@@ -0,0 +1,11 @@
+/* AT-79 Page Title - Level 1
+  This is the ResearchNow Variant, which isn't a semantic heading.*/
+
+// This story serves as documentation and example code for ResearchNow's Section Title.
+// Because ResearchNow uses React-Router's Link, the component itself must be passed in for routing to work properly.
+
+import * as React from "react";
+
+export default function RNSectionTitle(props: React.PropsWithChildren<{}>) {
```

Comment:
@EdwinGuzman 
> I see. If this should only be used in RN, then I don't think the Design System is the right place for it.

So this component (and its story) is gonna go against that whole philosophy.  I think there's a large conversation to be had here re: how to do project-specific overrides.  (Does the CSS for this component belong in the RN project?  I think there are obviously good arguments for "yes", and the argument for "no" is that having all the possibilities live somewhere in DS will make it harder for people to do stuff not explicitly blessed by DS, and possible to set up better automated testing).  Bigger conversation, right now project-specific overrides don't exist so ... it's a not-great-solution that we'll got to come back to later.

Reply:
Yeah, I'm all for going with this approach short-term. You're right that seeing specific examples might make it harder for others to not try something else.

Ideally, RN should not have to do overrides (or maybe just write minimal CSS) to create a component like this. I think a goal that should be aimed for later is that RN (and any other app) takes in both the styles and the react components and can compose them however it needs to and _not_ have to do overrides. But, yes, larger conversation and this might make more sense once the components are complete.

Reply:
I think a component like `Header-with-search` belongs in the DS. I think the UX team needs to iterate on this component more to get it into a more abstract state so that it's no longer RN-specific.